services/QuillLMS/app/models/concerns/public_progress_reports.rb

```diff
@@ -165,6 +166,8 @@ def formatted_score_obj(final_activity_session, activity, student)
       activity_classification_key = ActivityClassification.find(activity.activity_classification_id).key
       if ['lessons', 'diagnostic'].include?(activity_classification_key)
         score = get_average_score(formatted_concept_results)
+      elsif ['comprehension'].include?(activity_classification_key)
```

Comment:
I know we aren't doing it with the rest of these, but this should probably be `[ActivityClassification::COMPREHENSION_KEY].include?(activity_classification_key)`